langchain/src/vectorstores/elasticsearch.ts

```diff
@@ -38,7 +38,7 @@ export interface ElasticClientArgs {
 /**
  * Type representing a filter object in Elasticsearch.
  */
-type ElasticFilter = object;
+type ElasticFilter = { field: string, operator: string, value: any }[];
```

Comment:
This looks breaking? Shouldn't this allow `term` as a property?

Reply:
Hi jacoblee, thanks for taking your time to check this PR out.

With the `operator` property we can be more flexible and allow for "term", "terms", "range", "exists", etc. instead.
As I typed it as an array of objects, we can have multiple different filters applied.
The breaking part seems to be in the docs, but it seems I can't change those?

Reply:
Ah got it, yeah looks like it might have just been a placeholder.